include/rmm/cuda_device.hpp

```diff
@@ -52,7 +52,7 @@ namespace detail {
  */
 inline cuda_device_id current_device()
 {
-  int dev_id;
+  int dev_id{};
```

Comment:
This isn't a hill I will die on, but this is exactly the kind of example listed in the Exception section to CPP core guidelines "always initialize" rule: http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-always

Reply:
I don't understand why? In this case it's not expensive to zero the variable.

Reply:
> In this case it's not expensive to zero the variable.

Right, that's why it's not a hill I'll die on. I just wanted to point out that the CPP Core Guidelines do explicitly provide an exception for this kind of situation.